js/password.js

```diff
@@ -0,0 +1,22 @@
+(() => {
+
+	let input = document.getElementById("password");
+	let paragraph = document.getElementById("paragraph");
+
+	input.addEventListener("keydown", () => {
+		let password = input.value;
+		if (password.length < 10) {
+			input.style.backgroundColor = "red";
+			paragraph.textContent = "bad cat";
+		} else if (password.length < 18) {
+			input.style.backgroundColor = "orange";
+			paragraph.textContent = "good cat"
+		} else {
+			input.style.backgroundColor = "green";
+			paragraph.textContent = "Purrfect!"
+		}
+
```

Comment:
Could clear some of this white space to neaten it up but that is just being picky!
It looks great and all works.